README.md

```diff
@@ -1,4 +1,14 @@
-# build a current ffh gluon
+## well-known branches
+
+|    Branch Name   |       Description      |                Builds Against the Following Gluon Version                |            Used For           |
```

Comment:
I find it irritating to have the entry `manual builds` in every row;
but that's just DBS calling "Normalize the database", I'm afraid.
Fine with me.